iotqatools/cb_v2_utils.py

```diff
@@ -348,7 +346,10 @@ def __create_attributes(self, entity_context):
 
     def create_entities(self, context, entities_number, attributes_number):
```

Comment:
The context comes from step data in behave. See http://pythonhosted.org/behave/tutorial.html#step-data